### ISSUE TITLE
Shaping/Manipulating Solid Matter

### ISSUE THREAD

**Issue by raihje** [openlegend/core-rules#96]
Alright so, it's come up a few times that people would like to raise walls, cut fissures, etc into terrain, or shape existing constructs. I propose something like this, obviously it needs to be refined:

Manipulate Matter
PL: 5
Attributes: Creation, Alteration, Entropy
You can shape a 5' cube of matter, whether it's stone, earth, ice, or other solid matter.
This could mean changing a boulder into the shape of a fist, or raising the ground to form a wall.
If Creation is used, you can shape living matter.
If Entropy is used, you can shape corpses and other dead matter.
If Alteration is used, you can alter the weight of what you're shaping.

maybe split it into tiers, so that creative users aren't just melting a wall with entropy at first level or turning their dog into a living shield with creation.

**Comment by brianfeister** [comment]
Not a bad start - thanks for the proposal, we will do something like this.

**Comment by DukeAnax** [comment]
I had a discussion like this earlier in an Elemental Bender (Avatar series) thread, we took this as a direct attribute action, rather than a boon.  For starters we got something like a CR +1 for each 5' cube of earth or rock affected (it's supposed to be fairly easy in that world) a more difficult option could be CR +5 for each 5' cube of solid matter with a base CR of 10.

As a boon I'd do something like Resolute Barrier simmilar to Retributive Barrier, but instead of dealing damage it would prevent movement until it's defenses are overcome.

**Comment by raihje** [comment]
@brianfeister Genesis is PL 3 so I'd be tempted to make this the same PL since it's just manipulating what's already existing. The above makes sense to, attribute actions are somewhat defined I think..
I'd also make it a bit more reasonable to invoke, with 16 instead of 20 required for the roll.

**Comment by istabosz** [comment]
Thanks @raihje I'll take a crack at this.

**Comment by hmack312** [comment]
Couldn't you simply use the alteration skill and transmutation boon? With the Boon Focus II feat you can invoke it as a major action. I would consider allowing this to also make walls out of the ground when they hit power level 5 and can change the size or weight.

**Comment by brianfeister** [comment]
Not a bad idea @hmack312 but it seems reasonable to be capable of terrain shaping with a high Alteration score without needing to invest in a feat. That's my 2 cents anyway

**Comment by Munrock** [comment]
Would that 5' cube be the total volume of matter, or the bounds in which matter can be manipulated?

**Comment by brianfeister** [comment]
the area that you can influence, but let's not discuss this yet because I think there are a number of things we want to clean up before we make this an official first proposal.

**Comment by raihje** [comment]
IT'd overlap with @istabosz'z new Barrier boon, but I think this is much more versatile for simple things. Barrier is certainly one application of manipulating matter, one could also form an effigy of corpses and the like from Entropy, or perhaps intertwine branches or shape a dog's jaw into something monstrous with Creation. Fleshcrafting would be an interesting addition for darker campaigns like vampire the masquerade and the like where mind games and fears are played upon in spades.

**Comment by istabosz** [comment]
So, the more I think about this, the more I think it would better fit as just an attribute roll with a CR set by the GM based on the difficulty of what is being accomplished. The scope of effects possible when it comes to reshaping matter seems too vast to appropriately balance them within the constraints of a defined boon.

For example, being able to reshape a 5' cube means being able to shape a boulder into a fist, but it also means being able to get past basically any door or wall.

I kinda see it similar to smashing down a door or picking a lock. We don't need boons for that. We need a GM to decide how tough the task is based on the situation.

One section that I plan to add to chapter 2 is a vast assortment of examples of what could be done with different attribute checks so that GM and player feel more empowered to come up with these sorts of effects on the fly.

**Comment by Odog4ever** [comment]
@istabosz The more examples the better! Also I like a straight attribute roll and CR being the enabler of these more open abilities. At some point feat bloat is going to set in and we will need a feat/boon for lockpicking ;)

**Comment by brianfeister** [comment]
Fully agreed. In fact, I would say that we've probably already hit "feat / bane / boon bloat" and that we will need to make a very intentional effort to cull and kill as much as we can before we go to print. The recent additions and changes are all  good, but some of these things are really far-reaching. Change optional shapes of multi-target effects for example is going multiply the complexity and possibility array exponentially and I have a feeling we already have "bugs" we haven't found from the various combinations of new rules.

**Comment by brianfeister** [comment]
let's work on adding to the CR table with a bunch of these hypothetical scenarios

**Comment by istabosz** [comment]
Roger. I'm gonna make that my next task @brianfeister

**Comment by EliManthey** [comment]
don't wanna be nosy but was wondering the status of this stuff.

**Comment by istabosz** [comment]
@EliManthey It's been an ongoing project that I've been working on in between other updates. I've been drafting a large addition to chapter 2 that provides multi-genre examples of success with a twist and failure for every attribute at three different CRs. The example scenarios are specifically not tied to combat, banes, or boons in order to show the versatility of the attributes beyond the types of actions that are spelled out mechanically.

Almost done, though. I think I've just got Will and Protection left last time I checked.
